### PR TITLE
Added Cypress E2E test for GIBCT mobile

### PR DESCRIPTION
**Pull request by ksantiagoBAH**
## Description
As a developer, I need to create end to end tests for the mobile view of the comparison tool profile so that the functionality can be regression tested as new functionality is developed.
https://app.zenhub.com/workspaces/vft-59c95ae5fda7577a9b3184f8/issues/department-of-veterans-affairs/va.gov-team/9136

## Testing done
local testing using Cypress

`yarn cy:open`

select e2e to test

## Acceptance criteria
- [x] A mobile end to end test exists and passes. Tests cover the following searches:
a. VET TEC
b. Chapter 33 (to represent non-VET TEC)
- [x] The test covers the navigation required to reach the profile page (landing page and search results).
- [x] Axe checks are executed on the search results filters display on mobile.
- [x] Axe checks are executed on the profile page.
- [x] Axe checks are executed on the landing page.

## Definition of done
- [x] Events are logged appropriately
- [x] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/gi/tests/e2e/06-vet-tec-mobile.cypress.spec.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+  it('Default VETTEC profile flow with giBillChapter chapter 33', () => {
+    cy.viewport(481, 750);
+    // Landing Page
+    typeOfInstitution('vettec');
+    cy.axeCheck();
+
+    search();
+    cy.axeCheck();
+    expectLocation('/program-search');
```
Comment (zurbergram) [inline src/applications/gi/tests/e2e/06-vet-tec-mobile.cypress.spec.js:25]: with the version of the `search()` in this PR this `expectLocation` is not needed

File: src/applications/gi/tests/e2e/06-vet-tec-mobile.cypress.spec.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+        result.attributes.description
+      }`;
+      cy.get(createId(resultId)).should('be.visible');
+    });
+
+    const vetTecAttributes = vetTecSearchResults.data[0].attributes;
+    const profileLink = `/profile/${vetTecAttributes.facility_code}/${
+      vetTecAttributes.description
+    }`;
+
```
Comment (zurbergram) [inline src/applications/gi/tests/e2e/06-vet-tec-mobile.cypress.spec.js:39]: Does the filters overlay not occur on mobile for VET TEC Search Page?

File: src/applications/gi/tests/e2e/07-gibct-mobile.cypress.spec.js [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+  });
+
+  it('Default GIBCT mobile view profile flow with giBillChapter chapter 33', () => {
+    cy.viewport(481, 750);
+
+    // Landing Page
+    typeOfInstitution('school');
+    cy.axeCheck();
+
+    search('DODGE CITY COMMUNITY COLLEGE');
```
Comment (zurbergram) [inline src/applications/gi/tests/e2e/07-gibct-mobile.cypress.spec.js:23]:
```suggestion
    search(searchResults.data[0].attributes.institution);
```

File: src/applications/gi/tests/e2e/06-vet-tec-mobile.cypress.spec.js (hunk 2 above)
Comment (ksantiagomorgan6) [reply src/applications/gi/tests/e2e/06-vet-tec-mobile.cypress.spec.js:42]: updated

File: src/applications/gi/tests/e2e/06-vet-tec-mobile.cypress.spec.js (hunk 1 above)
Comment (ksantiagomorgan6) [reply src/applications/gi/tests/e2e/06-vet-tec-mobile.cypress.spec.js]: removed

**Review by zurbergram — APPROVED** [review]

**Review by sonntag-adam — APPROVED** [review]

**Pull request merged**